cypress/page-objects/signInPage.js

```diff
@@ -0,0 +1,43 @@
+class SignIn {
+  navigateToSignInPage() {
+    cy.visit('/signing');
+  }
+
+  typeSignInEmail(email) {
+    return cy.get('input[data-test="sign-in-email"]').type(email);
+  }
+
+  typeSignInPassword(password) {
+    return cy.get('input[data-test="sign-in-password"]').type(password);
+  }
+
+  clickSignInCTA() {
+    return cy.get('button').contains('Sign In').click();
+  }
+
+  typeSignUpName(name) {
+    return cy.get('.sc-citwmv input[type="text"]').type(name);
```

Comment:
we've discussed it before and please do use classes generated by `styled-components`
it is mostly my fault that they exist, but I don't have time to get rid of `styled-components`